ray/deployments/model.py

```diff
@@ -9,6 +9,7 @@
 from pymongo import MongoClient
 from ray import serve
 from ray.serve import Application
+from torch.cuda import max_memory_allocated, reset_peak_memory_stats
```

Comment:
Maybe look at this https://github.com/pytorch/torchtitan/blob/main/torchtitan/profiling.py

Reply:
I think for now it is a bit overkill (since I'm just fetching max gpu memory), but would be nice for capturing gpu/cpu memory traces.